### PR TITLE
Added NavBar placeholder

### PR DESCRIPTION
**Pull request by amead99**
NavBar file renders Home and Recipes links, and conditionally renders link to Account & Admin for admin, Account for user, and Login/Register when neither is true.

### CODE REVIEW

**Review by ljreaux — APPROVED** [review]

File: FE/src/components/NavBar.jsx
Comment (BarryFullStack) [inline FE/src/components/NavBar.jsx:1]: Could we add logout to the navbar in preparation for when it is a dropdown?

**Review by jared23m — APPROVED** [review]
approved

**Review by BarryFullStack — APPROVED** [review]

File: FE/src/App.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ function App() {
 
   return (
     <><h1>Hello Lab Rats</h1>
-    <NavBar />
+    <NavBar setToken={setToken}/>
```
Comment (BarryFullStack) [inline FE/src/App.jsx:18]: need to pass token as well.

File: FE/src/components/NavBar.jsx [hunk 2]
```diff
@@ -1,8 +1,44 @@
 import { Link } from 'react-router-dom'
+import { useNavigate } from 'react-router-dom'
+import { useState } from 'react'
 
-export default function NavBar ({token}) {
+export default function NavBar ({setToken}) {
```
Comment (BarryFullStack) [inline FE/src/components/NavBar.jsx:5]: also needs prop of token

**Pull request merged**